ovos_core/intent_services/ocp_service.py

```diff
@@ -104,6 +106,7 @@ def __init__(self, bus=None, config=None):
                          resources_dir=f"{dirname(__file__)}")
 
         self.ocp_api = OCPInterface(self.bus)
+        self.legacy_api = ClassicAudioServiceInterface(self.bus)
```

Comment:
If I read correctly, we only ever use one of `ocp_api` and `legacy_api` depending on `use_legacy_audio`. Consider only initializing this if `use_legacy_audio` is true

Reply:
could make it lazy, but the goal was to allow use_legacy_audio to change at runtime (if mycroft.conf changes)